From 0b4e72612d807caf22cfa9168cd69c18bd250d8c Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Tue, 4 Sep 2012 11:12:00 -0500
Subject: [PATCH] Revert "Revert "arm: armv7: add compile option
 -mno-unaligned-access if available""

This reverts commit c50afc1dab78a1a72bc37b60d97864e56b52cc5c.
---
 arch/arm/cpu/armv7/config.mk |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/cpu/armv7/config.mk b/arch/arm/cpu/armv7/config.mk
index 5407cb6..560c084 100644
--- a/arch/arm/cpu/armv7/config.mk
+++ b/arch/arm/cpu/armv7/config.mk
@@ -26,6 +26,8 @@ PLATFORM_RELFLAGS += -fno-common -ffixed-r8 -msoft-float
 # supported by more tool-chains
 PF_CPPFLAGS_ARMV7 := $(call cc-option, -march=armv7-a, -march=armv5)
 PLATFORM_CPPFLAGS += $(PF_CPPFLAGS_ARMV7)
+PF_CPPFLAGS_NO_UNALIGNED := $(call cc-option, -mno-unaligned-access,)
+PLATFORM_CPPFLAGS += $(PF_CPPFLAGS_NO_UNALIGNED)
 
 # =========================================================================
 #
-- 
1.7.10.4

